templates/_base-images-ref.html

```diff
@@ -0,0 +1,461 @@
+<script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+
+  (function () {
+    window.jQuery
+      .ajax({
+        url: "https://api.balena-cloud.com/v6/device_type?$select=name,slug&$expand=is_of__cpu_architecture($select=slug)&$filter=is_default_for__application/any(idfa:((idfa/is_host%20eq%20true)%20and%20(idfa/is_archived%20eq%20false))%20and%20not(contains(idfa/app_name,%27esr%27)))&$orderby=name%20asc"
+      })
+      .done(function (deviceTypes) {
+
+        var IGNORE_THESE_DEVICES = ['n310-tx2', 'apalis-imx6', 'colibri-imx6', 'cubox-i', 'edge', 'etcher-pro', 'edison', 'hummingboard2', 'odroid-u3+', 'odroid-ux3', 'nuc', 'parallella-hdmi-resin', 'solidrun-imx6']
```

Comment:
Heads-up that only the following are in the list of returned items: 'n310-tx2', 'etcher-pro'.
Any chance the rest are just aliases? eg `nuc` is an alias and the actual DT slug is `intel-nuc`
Here is the actual slug list:
```
["n310-tx2", "apalis-imx6q", "colibri-imx6dl", "hummingboard", "edge", "etcher-pro", "intel-edison", "hummingboard", "odroid-xu4", "odroid-xu4", "intel-nuc", "parallella", "hummingboard"]
```
which I got by doing
```
await Promise.all(['n310-tx2', 'apalis-imx6', 'colibri-imx6', 'cubox-i', 'edge', 'etcher-pro', 'edison', 'hummingboard2', 'odroid-u3+', 'odroid-ux3', 'nuc', 'parallella-hdmi-resin', 'solidrun-imx6'].map(dt => sdk.models.device.getDeviceSlug(dt)))
```

Now the extra question on top of this is why do we actually need to exclude those?
Probably better to add as a comment so that we don't forget.

Reply:
@thgreasi, I create that list based on this file https://docs.google.com/spreadsheets/d/1AX0r5P-JuL4tllFUJNEURYpgazrWQ6aq2Ac94Yev5Mg/edit#gid=1120827262, basically all devices which are not in our contracts, no longer supported will be in the list. I'm not sure what will be returned by the `device-type` endpoint so will list all the devices without base images in the list.

Reply:
@nghiant2710 it feels weird that we have device types that we support for which we don't have base images for. Anything holding us back from releases base images for those as well?
Discontinued DTs should not be returned by the above OData query, since we should be marking them as archived in the DB. 
I went through them and I think the only ones that are still in the list are the following:
```suggestion
        // Device Types for which we don't have base images
        var IGNORE_THESE_DEVICES = ["n310-tx2", "etcher-pro", "intel-edison", "odroid-xu4", "intel-nuc"];
```

Reply:
@thgreasi we don't create the base images for them because they're not in our contracts. I updated the list based on your suggestion (`intel-edison`, `intel-nuc` removed since they're in contracts)